spec/lib/govuk_logging_spec.rb

```diff
@@ -71,22 +86,60 @@ def app
         Rails.application
       end
 
-      it "logs errors thrown by the application" do
+      it "logs errors thrown by the application with govuk_request_id" do
+        stub_const("GdsApi::GovukHeaders", GovukHeaders)
         GovukLogging.configure
         get "/error"
-        $stderr.rewind
-        lines = $stderr.read.split("\n")
+        fake_stdout.rewind
+        lines = fake_stdout.read.split("\n")
         expect(lines).to include(/default exception/)
         error_log_line = lines.find { |log| log.match?(/default exception/) }
         expect(error_log_line).not_to be_empty
+
         error_log_json = JSON.parse(error_log_line)
         expect(error_log_json).to match(hash_including(
-                                          "exception_class" => "StandardError",
-                                          "exception_message" => "default exception",
+                                          "govuk_request_id" => "some-value",
                                         ))
-        expect(error_log_json).to have_key("stacktrace")
-        expect(error_log_json["stacktrace"]).to be_a(Array)
+
+        error_log_json_msg = error_log_json["message"]
+        expect(error_log_json_msg).to include("StandardError")
+        expect(error_log_json_msg).to include("default exception")
+        expect(error_log_json_msg).to match(/[a-zA-Z]+.*:[0-9]+:in.*/)
+      end
+
+      it "logs errors thrown by the application with no govuk_request_id" do
+        GovukLogging.configure
+        get "/error"
+        fake_stdout.rewind
+        lines = fake_stdout.read.split("\n")
+        expect(lines).to include(/default exception/)
+        error_log_line = lines.find { |log| log.match?(/default exception/) }
+        expect(error_log_line).not_to be_empty
+        error_log_json = JSON.parse(error_log_line)
+        error_log_json_msg = error_log_json["message"]
+        expect(error_log_json_msg).to include("StandardError")
+        expect(error_log_json_msg).to include("default exception")
+        expect(error_log_json_msg).to match(/[a-zA-Z].*:[0-9]+:in.*/)
+      end
+
+      it "logs to stdout in JSON format with govuk_request_id" do
+        stub_const("GdsApi::GovukHeaders", GovukHeaders)
```

Comment:
```suggestion
        stub_const("GdsApi::GovukHeaders", govuk_headers_class)
```

And skip it here too.